torch/export/__init__.py

```diff
@@ -252,6 +252,9 @@ def forward(self, x):
     """
     from torch._export import save
 
+    if not isinstance(ep, ExportedProgram):
+        raise TypeError("The 'ep' parameter must be an instance of 'ExportedProgram'.")
```

Comment:
Maybe include the type of `ep` in the error message as well? Like, expected this got that.

Reply:
Should be fine now. 🙂